### PR TITLE
Camera trigger update

### PR DESCRIPTION
**Pull request by birchera**
* Correct initialization of trigger pins
* Keep alive functionality specifically for Sony QX1 camera

Camera turn on/off is not enabled yet. Functionality is unclear right now. Will follow in a separate PR.

### CODE REVIEW

File: src/drivers/camera_trigger/camera_trigger.cpp [hunk 1]
```diff
@@ -410,6 +464,7 @@ CameraTrigger::cycle_trampoline(void *arg)
 
 				// Check that distance threshold is exceeded and the time between last shot is large enough
 				if ((trig->_last_shoot_position - current_position).length() >= trig->_distance) {
+					warnx("Shooting once, distance set to %2.2f", (double)trig->_distance);
```
Comment (LorenzMeier) [inline src/drivers/camera_trigger/camera_trigger.cpp]: Debug output?

File: src/drivers/camera_trigger/camera_trigger_params.c [hunk 2]
```diff
@@ -36,25 +36,12 @@
...
- * Selects the trigger interface
- *
- * @value 1 GPIO
- * @value 2 Seagull MAP2 (PWM)
- *
- * @reboot_required true
- *
- * @group Camera trigger
- */
-PARAM_DEFINE_INT32(TRIG_INTERFACE, 2);
```
Comment (LorenzMeier) [inline src/drivers/camera_trigger/camera_trigger_params.c]: Is this a bad merge?

Comment (birchera) [inline src/drivers/camera_trigger/camera_trigger_params.c]: you had it in twice

**Comment by LorenzMeier** [comment]
Thanks!

**Pull request merged**